src/main/java/seedu/multitasky/storage/XmlSerializableEntryBook.java

```diff
@@ -59,15 +59,11 @@ public XmlSerializableEntryBook(ReadOnlyEntryBook src) {
         this();
         events.addAll(src.getEventList().stream().map(XmlAdaptedEntry::new).collect(Collectors.toList()));
         deadlines.addAll(src.getDeadlineList().stream().map(XmlAdaptedEntry::new)
-                .collect(Collectors.toList()));
+                            .collect(Collectors.toList()));
         floatingTasks.addAll(src.getFloatingTaskList().stream().map(XmlAdaptedEntry::new)
-                .collect(Collectors.toList()));
-        archive.addAll(src.getArchive().stream().map(XmlAdaptedEntry::new)
-                .collect(Collectors.toList()));
-        bin.addAll(src.getBin().stream().map(XmlAdaptedEntry::new)
-                .collect(Collectors.toList()));
+                                .collect(Collectors.toList()));
```

Comment:
It's a line break indentation following this example in the Java Coding Standard doc:

![image](https://user-images.githubusercontent.com/29548741/28224421-89f31bb4-6901-11e7-8be7-bfed721e80d5.png)